var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -147,8 +147,8 @@ def install(self, spec, prefix):
                      % join_path(prefix, 'etc', 'openssl')]
         if spec.satisfies('platform=windows'):
             base_args.extend([
-                'CC=%s' % os.environ.get('CC'),
-                'CXX=%s' % os.environ.get('CXX'),
+                'CC=\"%s\"' % os.environ.get('CC'),
+                'CXX=\"%s\"' % os.environ.get('CXX'),
```

Comment:
I think this will start printing a warning:

> Quotes aren't needed because spack doesn't use a shell. Consider removing them.

I think we'll want to update the later call with `ignore_quotes=True` like

```
Executable('perl')('Configure', *base_args, ignore_quotes=True)
```

And I also think a comment would be useful here (i.e. the underlying Executable needs the quotes to parse this argument correctly, vs. this being a matter of inter-argument parsing).